[WFLY-19692]: Documentation for WFCORE-6960 read-config-as-xml-file

ehsavoie opened the pull request:
Updating CLI recipes for the new operation "read-config-as-xml-file".

Jira: https://issues.redhat.com/browse/WFLY-19692
      https://issues.redhat.com/browse/WFCORE-6960

fabiobrz commented on docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc:
```diff
@@ -190,14 +190,19 @@ Assume you have a host that is called "primary"
 
 [source,options="nowrap"]
 ----
-[domain@localhost:9990 /] /host=primary:read-config-as-xml
```
The new text should be added and the old one should NOT be removed now, since the `read-config-as-xml` still exists.

fabiobrz commented on docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc:
```diff
@@ -190,14 +190,19 @@ Assume you have a host that is called "primary"
 
 [source,options="nowrap"]
 ----
-[domain@localhost:9990 /] :read-config-as-xml
```
Same as above, let's add rather than replace.

fabiobrz commented on docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc:
```diff
@@ -190,14 +190,19 @@ Assume you have a host that is called "primary"
 [source,options="nowrap"]
 ----
-[domain@localhost:9990 /] :read-config-as-xml
+[domain@localhost:9990 /] attachment display --operation=:read-config-as-xml
```
According to the analysis document we'd actually need to use `read-config-as-xml-as-file`, right?

fabiobrz commented on docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc:
```diff
@@ -190,14 +190,19 @@ Assume you have a host that is called "primary"
+
+If you want to save the file you can use:
+----
+[standalone@localhost:9990 /] attachment save --operation=:read-config-as-xml --file=/home/wildfly/tmp/standalone-runtime.xml
```
According to the analysis document we'd actually need to use `read-config-as-xml-as-file`, right?

fabiobrz reviewed (changes requested):
Thanks for this PR @ehsavoie - I've dropped a couple of comments and suggestions.
Also please take the following one into account: https://docs.wildfly.org/33/Admin_Guide.html#operations here too the new operation should be mentioned.

fabiobrz commented on docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc:
```diff
@@ -215,6 +230,8 @@ Just for the domain or standalone
 }
 ----
 
+Please note that the operation"
```
Leftover? It seems the sentence which is started by these words has been added above already.

fabiobrz reviewed (changes requested):
Thanks @ehsavoie for addressing my remarks.
I am ok about not advertising `read-config-as-xml` and the deprecation note you added.
I added just one comment for what seems to be a leftover typo.

Also you might want to update the PR title in order to have it pass the CI checks, see https://github.com/wildfly/wildfly/pull/18159#issuecomment-2320730648

ehsavoie committed:
[WFLY-19692]: Documentation for WFCORE-6960  read-config-as-xml-file

Updating CLI recipes for the new operation "read-config-as-xml-file".

Jira: https://issues.redhat.com/browse/WFLY-19692
      https://issues.redhat.com/browse/WFCORE-6960

Signed-off-by: Emmanuel Hugonnet <ehugonne@redhat.com>

fabiobrz reviewed (approved):
Thanks @ehsavoie, for fixing the typos. Changes LGTM, approving.

bstansberry commented:
Thanks @ehsavoie and @fabiobrz
